mtags/src/main/scala-2/scala/meta/internal/pc/SignatureHelpProvider.scala

```diff
@@ -171,7 +171,11 @@ class SignatureHelpProvider(val compiler: MetalsGlobal)(implicit
           o.info
             .member(compiler.nme.apply)
             .safeAlternatives
-            .map(alt => alt -> qual.tpe.memberType(alt))
+            .flatMap { alt =>
+              val tpe = qual.tpe
+              if (tpe == null) None
+              else Some(alt -> tpe.memberType(alt))
+            }
```

Comment:
I woder it'd make sense to copy logic from line 181
```
          val recieverTpe = qual match {
            case Select(qualifier, _) if qualifier.tpe ne null => qualifier.tpe
            case _ => m.owner.info
          }
```

Reply:
It's all a bit different, I don't see an easy way to deduplicate. For methodSymbol we use the qualifier type from select, but for ModuleSymbol we just do .info. Looks like there is some refactoring potential, but it seems much broader scope than I intended 😅 .